state.go

```diff
@@ -388,6 +341,39 @@ func (o *OverflowState) Address(key string) string {
 	return fmt.Sprintf("0x%s", o.FlowAddress(key))
 }
 
+// return the flow Address of the given name
+func (o *OverflowState) FlowAddressE(key string) (*flow.Address, error) {
```

Comment:
what does the E stand for?

Reply:
Since golang does not support overloading i tend to just add the E suffix at the end of a method that returns whatever the original method does and an Error.

Reply:
ooooh right